CMakeLists.txt

```diff
@@ -263,6 +263,72 @@ ADD_CUSTOM_TARGET(indent
   COMMENT "Indenting all ASPECT header and source files..."
   )
 
+IF(CMAKE_GENERATOR MATCHES "Ninja")
+  SET(_make_command "$ ninja")
+ELSE()
+  SET(_make_command " $ make")
```

Comment:
```suggestion
  SET(_make_command "$ make")
```

Reply:
it is made such that both commands have the same width (so that the info message is aligned nicely without a double space between ``make`` and ``release`` for example).

Reply:
isn't there a cmake variable that spells out the name of the generator? I thought @tamiko used it in the deal.II patch...

Reply:
I'm saying this because there may be other generators we could use, e.g. the MS VC internal one at one point.

Reply:
This is copied verbatim from deal.II:
https://github.com/dealii/dealii/blob/d9bf195abe45827205522a31094d814ad416c4fd/cmake/macros/macro_deal_ii_invoke_autopilot.cmake#L41